open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/application.properties

```diff
@@ -71,3 +71,10 @@ logging.level.root=ERROR
 logging.level.org.springframework=ERROR
 logging.level.org.springframework.boot.web.embedded.tomcat=INFO
 logging.level.org.odpi.openmetadata=INFO
+
+################################################
+### Zuul
+################################################
+zuul.ignoredPatterns=/api/**,/error/**
+zuul.routes.ui.path=/**
+zuul.routes.ui.url=http://localhost:8080
```

Comment:
> Also how does the static server get launched if a user just builds this project

`npm install` in `odpi/egeria-ui` and `npm run serve` and it will start on http://localhost:8080

Reply:
> As identified in call we would be good to make this https - as we've already self-certified as using secure protocols for egeria?

This can be passed as an ENV variable at deploy time, for development purposes it can remain http:// as de development server in egeria-ui is http://